src/nft/components/collection/FilterButton.tsx

```diff
@@ -16,6 +17,8 @@ export const FilterButton = ({
   onClick: () => void
   collectionCount?: number
 }) => {
+  const hideResultsCount = window.innerWidth >= breakpoints.sm && window.innerWidth < breakpoints.md
```

Comment:
You can use the isMobile hook here

Reply:
no this is different from isMobile

Reply:
isMobile is a different case, this one is testing for medium sized ish screens.